### PR TITLE
lnav 0.10.1

### PR DESCRIPTION
**Pull request by easp**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/lnav.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Lnav < Formula
   desc "Curses-based tool for viewing and analyzing log files"
   homepage "https://lnav.org/"
-  url "https://github.com/tstack/lnav/releases/download/v0.10.0/lnav-0.10.0.tar.gz"
-  sha256 "05caf14d410a3912ef9093773aec321e0f4718a29476005c05dd53fcd6de1531"
+  url "https://github.com/tstack/lnav/releases/download/v0.10.1/lnav-0.10.1.tar.gz"
+  sha256 "a1fd65916bf06e5f01f07aca73ff9cca783f0562465facdf28fa24e9cf568209"
   license "BSD-2-Clause"
   revision 1
```
Comment (carlocab) [inline Formula/lnav.rb:7]:
```suggestion
```
`revision` lines are removed at version bumps.

Comment (easp) [reply Formula/lnav.rb]: Thanks. Fixed, committed and pushed.

**Review by Bo98 — APPROVED** [review]
Thanks!

**Comment by black-snow** [comment]
0.10.1 errors out for me

> ==> ./configure --prefix=/home/linuxbrew/.linuxbrew/Cellar/lnav/0.10.1 --libdir=/home/linuxbrew/.linuxbrew/Cellar/lnav/0.10.1/lib --with-sqlite3=/home/linuxbrew/.linuxbrew/opt/sqlite --with-readline=/home/linuxbrew/.linuxbrew/opt/readl
==> make install V=1
Last 15 lines from /home/rbergmann/.cache/Homebrew/Logs/lnav/02.make:
  199 |     std::fill_n(std::ostream_iterator<std::string>(os), num, input);
      |                      ^~~~~~~~~~~~~~~~
string_util.cc:38:1: note: 'std::ostream_iterator' is defined in header '<iterator>'; did you forget to '#include <iterator>'?
   37 | #include "string_util.hh"
  +++ |+#include <iterator>
   38 |
string_util.cc:199:50: error: expected primary-expression before '>' token
  199 |     std::fill_n(std::ostream_iterator<std::string>(os), num, input);
      |                                                  ^
make[2]: *** [Makefile:792: string_util.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: Leaving directory '/tmp/lnav-20220826-14681-7k352r/lnav-0.10.1/src/base'
make[1]: *** [Makefile:1358: install-recursive] Error 1
make[1]: Leaving directory '/tmp/lnav-20220826-14681-7k352r/lnav-0.10.1/src'
make: *** [Makefile:510: install-recursive] Error 1

**Pull request closed without merging**